common/version.go

```diff
@@ -39,9 +39,11 @@ import (
 	"strings"
 )
 
-const Blobfuse2ListContainerURL = "https://blobfuse2.blob.core.windows.net/release"
-
-const BlobFuse2WarningsURL = "https://github.com/Azure/azure-storage-fuse/wiki/9.-Security-Warnings"
+const (
+	Blobfuse2LatestReleaseURL      = "https://github.com/Azure/azure-storage-fuse/releases/latest"
+	Blobfuse2HasSecurityWarningURL = "https://github.com/Azure/azure-storage-fuse/securitywarnings/"
```

Comment:
This should be either be one of these:
1. `https://github.com/Azure/azure-storage-fuse/blob/main/securitywarnings/`
2. `https://raw.githubusercontent.com/Azure/azure-storage-fuse/main/securitywarnings/`